### ISSUE TITLE
Arms are weird

### ISSUE THREAD

**Issue by rachelleeann** [tim-elmer/feminine-female-farmer#4]
When you update your characters hair using the mirror from fashion sense the arms stick out through the hair, meaning you can see the arms from behind

**Comment by tim-elmer** [comment]
Interesting. Does the issue persist when you leave the mirror UI? Can you provide a screenshot (I don't play with this mod enabled, or with Fashion Sense)?

**Comment by rachelleeann** [comment]
Before
![Screenshot 3_9_2023 9_28_09 PM](https://user-images.githubusercontent.com/127448674/224208189-e93c2cad-cc88-47c9-a3b3-f8bee65aab21.png)
After
![image](https://user-images.githubusercontent.com/127448674/224208358-a324a60a-5bfe-4df7-b055-3ced5d76c5cf.png)
After running
![image](https://user-images.githubusercontent.com/127448674/224208458-6901ad1a-32fc-40c6-b910-a45a19e25c7d.png)
In mirror (Fashion Sense)
![image](https://user-images.githubusercontent.com/127448674/224208519-a8879ba3-e57f-4553-8218-9d8d7cfe9912.png)

**Comment by tim-elmer** [comment]
Hmm. Unfortunately I think this may be related to how Fashion Sense works its magic. As this mod is just a Content Patcher patch, I don't think there's much - if anything - that I can do.

Does the issue persist after restarting the game?

**Comment by rachelleeann** [comment]
So I player tested a bit more and it seems like the mod isn't registering that there's still hair on the sprite. It only looks bad when I apply long hair. 

There's another mod very similar to this one on Nexus, More Elegant Farmer Body, that works with FS. Makes me hope that this one could, too

**Comment by tim-elmer** [comment]
The thing is, this mod doesn't "know" _anything_ about hair. All this mod (and by extension Content Patcher) does is edit the sprite sheet being loaded for the player base and shirts. That said, it shouldn't be rendering over the hair regardless. It's possible that the way I am loading the sprite sheet is causing compatibility issues with something else you have installed.

I can't quite reproduce the issue (using [Fashion Sense](https://www.nexusmods.com/stardewvalley/mods/9969) and [Shardust's Animated Hairstyles](https://www.nexusmods.com/stardewvalley/mods/10294) [I believe that's what you're using]) either way:

![image](https://user-images.githubusercontent.com/12878113/224431416-0e4a64e6-b9d4-48e0-a421-907e5a9a3af4.png)
![image](https://user-images.githubusercontent.com/12878113/224431600-7edd7f34-1476-48f2-874f-c95ab7dfbf7b.png)

That said, I switched to editing the sheet rather than loading a new image. Does this work how you expect it to? [feminine-female-farmer_edit-body-test.zip](https://github.com/tim-elmer/feminine-female-farmer/files/10946154/feminine-female-farmer_edit-body-test.zip)

**Comment by rachelleeann** [comment]
I'll check that out when I get the chance, I've been using the mod anyways because I figured I could either NOT change the hair or just use the short styles. But somehow, because I haven't changed anything, the mod is working fine. Any hair, short or long, it's fine. Super weird.

**Comment by rachelleeann** [comment]
I just had a thought, Fashion Sense updated today, I wonder if there was a fix??

**Comment by tim-elmer** [comment]
That's entirely possible, I just grabbed the latest version. 🤷‍♂️

**Comment by tim-elmer** [comment]
Completely unrelated; lovely farmhouse! :)

**Comment by rachelleeann** [comment]
![image](https://user-images.githubusercontent.com/127448674/224436787-36c4c55d-419d-4501-91e2-8f74291df5bb.png)

Screenshot of my log with your test file loaded

**Comment by tim-elmer** [comment]
Ah. I seem to have left a bunch of extra space on the right side from some of FFF's legacy sprite indices. Not sure why you got an error message and I didn't, but this should fix it.
[feminine-female-farmer_math-is-hard.zip](https://github.com/tim-elmer/feminine-female-farmer/files/10946689/feminine-female-farmer_math-is-hard.zip)

**Comment by rachelleeann** [comment]
Sadly it didn't work. But once you change the clothes the arms go back to normal. I'm gonna play with the mod either way lol, so thank you for the work you've put into it. ALSO, the farm house is: https://www.nexusmods.com/stardewvalley/mods/14411

Sidenote, do you know if this mod is multiplayer compatible?

**Comment by tim-elmer** [comment]
It should be, cosmetic mods will work just fine in multiplayer, but will only be visible for those using them.

**Comment by rachelleeann** [comment]
![image](https://user-images.githubusercontent.com/127448674/225815864-e153066e-4b19-42b8-9a3d-bb66e73f910d.png)
Come across this in the items menu, seems I'm missing quite a few base game shirts. Do you have any ideas how I could fix this? I'm fairly certain it's cause of the body mod

**Comment by tim-elmer** [comment]
Does the same thing happen with this mod disabled?

**Comment by rachelleeann** [comment]
No, it doesn't. Just checked. I'm also not running any other mods that change base game clothes.

**Comment by tim-elmer** [comment]
Yeah, thinking about it more, that makes sense. Because the original mod was created before the additional clothing items existed, there is just transparent space in those sprite indices. Unfortunately that's not something that I have the bandwidth to fix right now, as all of the vanilla shirt sprites not covered by the mod would need to be tweaked to fit.
